boot/zephyr/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -64,3 +66,11 @@ CONFIG_RESET_ON_FATAL_ERROR=n
 CONFIG_SPI=n
 CONFIG_I2C=n
 CONFIG_UART_NRFX=n
+
+# The following configurations are required to support simultaneous multi image update
+CONFIG_PCD_APP=y
+CONFIG_UPDATEABLE_IMAGE_NUMBER=2
```

Comment:
should thingy53 always be doing simultaneously  multi image updates ?

Reply:
I think that this should always be supported by the bootloader, because if network core is in use we may need to update it at some point.  Updating only application core is not enough in my opinion.

Reply:
I agree with @MarekPieta here, multi-image should be default for any device that may required update of additional cores.

Reply:
that sounds like something that should be present in nrf5340dk_nrf5340_cpu also then ?

Reply:
I am not sure if we are introducing support for this board now. I think the focus is only on Thingy:53.